k9mail/src/main/res/values/strings.xml

```diff
@@ -1192,33 +1192,42 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="openpgp_dialog_sign_only_minus_attach">Signatures may be displayed as \'signature.asc\' attachments in some clients.</string>
     <string name="openpgp_dialog_sign_only_neutral_encsigned">Encrypted messages always include a signature.</string>
 
-    <string name="crypto_msg_disabled">Message is not encrypted</string>
-
-    <string name="crypto_msg_encrypted_error">Message is encrypted, but there was a decryption error.</string>
-    <string name="crypto_msg_encrypted_insecure">Message is encrypted but insecure!</string>
-    <string name="crypto_msg_signed_error">Message is signed, but the signature could not be verified.</string>
-    <string name="crypto_msg_encrypted_no_provider">Message is encrypted, but no crypto app is configured.</string>
-    <string name="crypto_msg_unsupported_encrypted">Message is encrypted, but in an unsupported format.</string>
-    <string name="crypto_msg_unsupported_signed">Message is signed, but in an unsupported format.</string>
-    <string name="crypto_msg_incomplete_encrypted">Message is encrypted, but must be fully downloaded for decryption.</string>
-    <string name="crypto_msg_cancelled">Message is encrypted, but decryption was cancelled.</string>
-
-    <string name="crypto_msg_signed_unencrypted">Message is signed, but not encrypted.</string>
-    <string name="crypto_msg_encrypted_unsigned">Message was encrypted, but not signed.</string>
-
-    <string name="crypto_msg_sign_unknown">Signature made by unknown key.</string>
-    <string name="crypto_msg_sign_verified">Signature made by trusted key.</string>
-    <string name="crypto_msg_sign_unverified">Signature made by untrusted key!</string>
-    <string name="crypto_msg_sign_error">Signature contained an error.</string>
-    <string name="crypto_msg_sign_mismatch">Signature made by wrong key.</string>
-    <string name="crypto_msg_sign_expired">Signature made by expired key.</string>
-    <string name="crypto_msg_sign_revoked">Signature made by revoked key.</string>
-    <string name="crypto_msg_sign_insecure">Signature made by insecure key.</string>
-    <string name="crypto_msg_sign_incomplete">Message must be fully downloaded to verify its signature.</string>
-
-    <string name="crypto_msg_signed_encrypted">Message is signed and encrypted.</string>
-    <string name="crypto_msg_unsigned_encrypted">There is no signature, this message may have been intercepted.</string>
-
+    <string name="crypto_msg_title_plaintext">Plaintext</string>
+    <string name="crypto_msg_unencrypted_sign_error">End-to-End signature contained an error</string>
+    <string name="crypto_msg_incomplete_signed">must fully download to process End-to-End signature</string>
+    <string name="crypto_msg_unsupported_signed">contains unsupported End-to-End signature</string>
+
+    <string name="crypto_msg_title_unencrypted_signed_e2e">End-to-End signed plaintext</string>
```

Comment:
Technical note - when do we use E2E vs non E2E?

Reply:
The classification is that encrypted messages that don't have a valid signature are not "end-to-end", in particular messages without a signature are not end-to-end.

Reply:
Oh right, so you're only saying E2E in the happy case - then everything else is some level of imperfect. Makes sense